### PR TITLE
Data cookbooks update 2

### PR DESCRIPTION
**Pull request by Jkovarik**

### CODE REVIEW

File: docs/data-cookbooks/cnm-workflow.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ Prior to working through this entry you should be familiar with the [Cloud Notif
 
 ### Cumulus
 
-This entry assumes you have a deployed instance of Cumulus (>= version 1.8).
+This entry assumes you have a deployed instance of Cumulus (version >= 1.16.0).   The entry assumes you are deploying cumulus via the [`cumulus` terraform module](https://github.com/nasa/cumulus/tree/master/tf-modules/cumulus) sourced from the [release page](https://github.com/nasa/cumulus/releases).
```
Comment (markdboyd) [inline docs/data-cookbooks/cnm-workflow.md]:
```suggestion
This entry assumes you have a deployed instance of Cumulus (version >= 1.16.0).   The entry assumes you are deploying Cumulus via the [`cumulus` terraform module](https://github.com/nasa/cumulus/tree/master/tf-modules/cumulus) sourced from the [release page](https://github.com/nasa/cumulus/releases).
```

File: docs/data-cookbooks/cnm-workflow.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ You should be able to quickly use the "Create Data Stream" button on the [Kinesi
 
 ![Screenshot of AWS console page for creating a Kinesis stream](assets/cnm_create_kinesis_stream.jpg)
 
-Please bear in mind that your `{{prefix}}-lambda-processing` IAM role will need permissions to write to the response stream for this workflow to succeed if you create the Kinesis stream with a dashboard user.   If you are using the example deployment (or a deployment based on it), the IAM permissions should be set properly.
+Please bear in mind that your `{{prefix}}-lambda-processing` IAM role will need permissions to write to the response stream for this workflow to succeed if you create the Kinesis stream with a dashboard user.   If you are using the `cumulus` top-level module for your deployment this should be set properly.
```
Comment (markdboyd) [inline docs/data-cookbooks/cnm-workflow.md]: This role still exists/is relevant?

File: docs/data-cookbooks/cnm-workflow.md [hunk 3]
```diff
@@ -68,137 +68,171 @@ Provided the prerequisites have been fulfilled, you can begin adding the needed
...
 
-The following tasks are required to be defined in the `lambdas.yml` configuration file.
+To execute this workflow, you're required to include several lambda resources in your deployment.  To do this, add the following task (lambda) definitions to your deployment along with the workflow you created above:
 
-If you're using a deployment based on the [example deployment](https://github.com/nasa/cumulus/tree/master/example) these lambdas should already be defined for you.
+* https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/cnm_to_cma_task.tf
+* https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/cnm_response_task.tf
 
-#### CNMToCMA
+**Please note:** To utilize these tasks you need to ensure you have a compatible CMA layer.  See the [deployment instructions](../deployment/README.md) for more details on how to deploy a CMA layer.
```
Comment (markdboyd) [inline docs/data-cookbooks/cnm-workflow.md]: Cant we link to the exact header here?

File: docs/data-cookbooks/cnm-workflow.md [hunk 4]
```diff
@@ -68,137 +68,171 @@ Provided the prerequisites have been fulfilled, you can begin adding the needed
...
+JSON
 ```
 
 Again, please make sure to modify the value CNMResponseStream to match the stream name (not ARN) for your Kinesis response stream.
 
-### Task Configuration
+### Lambda Configuration
 
-The following tasks are required to be defined in the `lambdas.yml` configuration file.
+To execute this workflow, you're required to include several lambda resources in your deployment.  To do this, add the following task (lambda) definitions to your deployment along with the workflow you created above:
```
Comment (markdboyd) [inline docs/data-cookbooks/cnm-workflow.md]: Nit: can we change `lambda` to `Lambda` throughout?

File: docs/data-cookbooks/cnm-workflow.md (hunk 2 above)
Comment (Jkovarik) [reply docs/data-cookbooks/cnm-workflow.md]: Yes, I think so: https://github.com/nasa/cumulus/blob/master/tf-modules/ingest/iam.tf

File: docs/data-cookbooks/cnm-workflow.md (hunk 3 above)
Comment (Jkovarik) [reply docs/data-cookbooks/cnm-workflow.md]: Sure, when this was written I wasn't sure what that would be.

File: docs/data-cookbooks/cnm-workflow.md (hunk 4 above)
Comment (Jkovarik) [reply docs/data-cookbooks/cnm-workflow.md]: not a nit. 
b3cfd9534

File: docs/data-cookbooks/cnm-workflow.md (hunk 3 above)
Comment (Jkovarik) [reply docs/data-cookbooks/cnm-workflow.md]: 826ca9ec3

**Pull request merged**